client/app/services/list-configuration.service.js

```diff
@@ -0,0 +1,40 @@
+/* eslint camelcase: "off" */
+(function() {
+  'use strict';
+
+  angular.module('app.services')
+    .factory('ListConfiguration', ListConfigurationFactory);
```

Comment:
Hi, couldn't these helper functions been added to the already existing ListView.js?